test/inc/WindowsAppRuntime.Test.TAEF.cppwinrt.h

```diff
@@ -1,9 +1,13 @@
-﻿// Copyright (c) Microsoft Corporation and Contributors. All rights reserved.
+// Copyright (c) Microsoft Corporation and Contributors. All rights reserved.
 // Licensed under the MIT License.
 
 #ifndef __WINDOWSAPPRUNTIME_TEST_TAEF_CPPWINRT_H
 #define __WINDOWSAPPRUNTIME_TEST_TAEF_CPPWINRT_H
 
+#include <WexTestClass.h>
+#include <winrt/base.h>
+#include <windows.h>
```

Comment:
Order's backwards. General ordering would be

* windows
* C++/WinRT
* TAEF

(assuming any of this is desirable)